### ISSUE TITLE
Have different worksheets for different roles

### ISSUE THREAD

**Issue by acdalal** [StoneyJackson/github-workflow-activity#21]
It was confusing working between two worksheets, one with (sort of) instructions and one with commands. There was also a TON of information on both sheets. It might be easier to follow along and figure out what each person is supposed to be doing if there was a different worksheet with *just* the instructions for each particular role. So, for instance, the maintainer would get the "maintainer" worksheet with just the "maintainer" commands. When you are waiting for someone else to complete something, your worksheet can indicate this.

You could always make the full set of worksheets available to students at the end of the exercise.

**Comment by StoneyJackson** [comment]
This is great feedback. Thanks!

Before fixing anything, I would like to clarify and verify the problem. I'm hearing three issues:

1. Having to work with two worksheets is confusing and annoying.
2. There is way to much to read.
3. It's hard to follow a role.

Does this sound right? Please feel free to add, remove, and/or adjust.

**Comment by acdalal** [comment]
Yes, those 3 issues are exactly what I meant. Thanks!

**Comment by StoneyJackson** [comment]
Cool... now let's explore some solutions:

Solving R1:
1. One script per role with all commands inlined (as suggested by @acdalal).
2. One script with all roles and all commands inlined. Let's call this a "play".
(I thought about hyperlinking, but that can get equally confusing as you flip between pages. So I ruled it out.)

Solving R2:
1. Judicious editing?
2. Inlining commands could be customized for each situation, removing extra steps that might be needed in general.
3. Maybe this activity is just too big and should be split into separate activities. Then each activity would have a more manageable amount of text.

Solving R3:
1. A script per role with inlined commands.
2. A single script with all roles and commands inlined: a "play".
(I thought about hyperlinking, but that can get equally confusing as you flip between pages. So I ruled it out.)


Other solutions to these?


It looks like there are two major solutions to R1 and R3. A little pro-con to decide between these is probably in order.

For R2, we might want to analyze how much editing is possible to reduce the text. And if that doesn't look like it will buy us much, then we think about splitting into smaller activities.
